.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
```suggestion
.idea
```

Reply:
if you want but by convention folders should be preceded with a /

Reply:
no, it's just the directory separator.
Here you are only ignoring the folder at the root, that why you had to had the ignore in the front directory too.

Remove the slash or replace it with `**/.idea`, this should work too